read-fonts/src/font_data.rs

```diff
@@ -278,6 +278,12 @@ impl AsRef<[u8]> for FontData<'_> {
     }
 }
 
+impl<'a> From<&'a [u8]> for FontData<'a> {
```

Comment:
This is a useful addition generally, and hopefully fixes some of the ergonomics issues around needing to manually create `FontData`?

Reply:
+1. We could additionally change `FontRead::read` to take `impl Into<FontData>` for further convenience but that doesn’t really seem to be necessary.